test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,40 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+function getUnassignedPort(cb) {
+  const server = net.createServer();
+  server.listen({port: 0}, common.mustCall(() => {
+    // When the server is closed this port will no longer be assigned
+    const unassignedPort = server.address().port;
+    server.close(common.mustCall(() => {
```

Comment:
Funny story: I've been looking at this issue's brother #12951.
These tests are run in parallel first this one, then the other.
In the other one there's a server that's supposed to receive 6 connections, instead it received 7.
I wonder where that 7th request comes from 🤣